packages/eslint-plugin/rules.d.ts

```diff
@@ -75,5 +75,4 @@ export type TypeScriptESLintRules = Record<
 >;
 
 declare const rules: TypeScriptESLintRules;
-// eslint-disable-next-line import/no-default-export
```

Comment:
this is an unnecessary suppression reported by ESLint 9. It is unnecessary because this rule is already disabled on all `d.ts` files in our eslint config file